### PR TITLE
Update ConsumeNightlyDocs

### LINKED ISSUE #1105: Make docs for consuming nightly more generic
The docs for consuming the nightly build have a fixed preview version. This should be more generic to avoid misunderstandings.

[https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/ConsumeNightlyBuild.md](https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/ConsumeNightlyBuild.md)

>Yep we should remove the real version and add something like  `xxxx`

_Originally posted by @MarcoRossignoli in https://github.com/coverlet-coverage/coverlet/issues/1042#issuecomment-789080342_

### PR DESCRIPTION
**Pull request by daveMueller**
closes #1105

### CODE REVIEW

File: Documentation/ConsumeNightlyBuild.md [hunk 1]
```diff
@@ -16,28 +16,38 @@ To consume nightly builds, create a `NuGet.Config` in your root solution directo
...
 ```
 
-### Install packages
+## Install packages
 
-Visual Studio:
+### Visual Studio:
 
 ![File](images/nightly.PNG)
+Example:
```
Comment (MarcoRossignoli) [inline Documentation/ConsumeNightlyBuild.md:24]: new line needed?

image

**Review by MarcoRossignoli — APPROVED** [review]
Small nit but here we go thx!

File: Documentation/ConsumeNightlyBuild.md (hunk 1 above)
Comment (daveMueller) [reply Documentation/ConsumeNightlyBuild.md]: I now forced the line break.

image

**Pull request merged**